platform/broadcom/sonic-platform-modules-cel/dx010/modules/dx010-fan-led.c

```diff
@@ -0,0 +1,209 @@
+/*
+ * dx010-fan-led.c - Kernel module for Seastone DX010 fan LEDs
```

Comment:
the fan led control should be left to user space, can you revert this dx010-fan-led change?

Reply:
@pphuchar Please help to check this one.

Reply:
current ask is just revert this to previous version, we can deal with the user-space fan led control later. Not in this PR.